docx/oxml/__init__.py

```diff
@@ -200,3 +200,7 @@ def OxmlElement(nsptag_str, attrs=None, nsdecls=None):
 register_element_cls('w:br', CT_Br)
 register_element_cls('w:r',  CT_R)
 register_element_cls('w:t',  CT_Text)
+
+from .bookmark import CT_bookmarkStart, CT_bookmarkEnd
+register_element_cls('w:bookmarkStart', CT_bookmarkStart)
+register_element_cls('w:bookmarkEnd', CT_bookmarkEnd)
```

Comment:
* These "registrations" are in alphabetical order, to ease finding them later, which is a common occurrence. The "first key" is the module (`from .bookmark ...`) so these belong up with the 'b's.
* I assume you got the spelling remedied later, this is an example of where I can't easily tell because it may be in a later commit.
* The imports are also made to appear in alphabetical order (case and leading underscore insensitive). So they should appear as CT_BookmarkEnd, CT_BookmarkStart (reverse of current order).
* The `register_element_cls()` calls also appear in alphabetical order (by tag name), which eases detection of duplicates.
* This might give you an idea of the level to which conventions have established in this project. There are a lot of them you'll learn along the way. Together they make for rapid navigation and detection of errors. I know they can seem excessive to some folks, depending on one's disposition for extreme order :)